app/src/components/misc/SupportPanel/index.tsx

```diff
@@ -0,0 +1,196 @@
+                  trackSupportOptionClicked("chat_with_us");
+                } else {
+                  dispatch(
+                    // @ts-ignore
+                    actions.toggleActiveModal({
+                      modalName: "pricingModal",
+                      newValue: true,
+                      newProps: { selectedPlan: null, source: "support_option_clicked" },
+                    })
+                  );
+                  trackSupportOptionClicked("upgrade_to_chat_with_us");
+                }
+                paidUser
+                  ? window.$crisp.push(["do", "chat:open"])
+                  : dispatch(
+                      // @ts-ignore
+                      actions.toggleActiveModal({
+                        modalName: "pricingModal",
+                        newValue: true,
+                        newProps: { selectedPlan: null, source: "support_option_clicked" },
+                      })
+                    );
+              }}
+            >
+              <span className="support-panel-label-style ">
+                Chat With Us
+                {!paidUser && <span className="support-panel-upgrade-badge-style">Upgrade</span>}
+              </span>
+            </div>
+          ),
+          icon: <BsFillChatLeftTextFill className="support-panel-icon-style" />,
+        },
+
+        !slackInviteVisibilityStatus &&
+          user.loggedIn &&
+          !paidUser && {
+            key: "5",
+            label: (
+              <a
+                href="mailto:contact@requestly.io"
+                target="_blank"
+                rel="noreferrer"
+                className="support-panel-item-style"
+                onClick={() => {
+                  setVisible(false);
+                  trackSupportOptionClicked("send_message");
+                }}
+              >
+                <span className="support-panel-label-style ">Send Message</span>
+              </a>
+            ),
+            icon: <TbMailForward className="support-panel-icon-style" />,
+          },
+      ].filter(Boolean),
+    [dispatch, paidUser, slackConnect, slackInviteVisibilityStatus, user.loggedIn]
+  );
+  return (
+    <div className="support-panel-container">
+      <Dropdown
+        menu={{ items: supportItems }}
+        trigger={["click"]}
+        onOpenChange={(state) => setVisible(state)}
+        dropdownRender={(menu) => {
+          return <div className="support-panel-dropown-container">{menu}</div>;
+        }}
+      >
+        <Badge dot={!isSupportChatOpened} size="small">
+          <Button
+            type="primary"
+            size="large"
+            shape="circle"
+            icon={
+              visible || isRequestBotVisible ? (
+                <CloseOutlined />
+              ) : (
+                <PiChatTeardropTextFill className="support-panel-icon-style " />
+              )
+            }
+            onClick={() => {
+              if (visible) window?.$crisp.push(["do", "chat:close"]);
```

Comment:
https://github.com/requestly/requestly/pull/1968#discussion_r1684086883
```suggestion
              if (visible) window?.$crisp?.push(["do", "chat:close"]);
```